pkg/daemon/ovs_windows.go

```diff
@@ -318,7 +318,7 @@ func configureMirrorLink(portName string, mtu int) error {
 	return nil
 }
 
-func configProviderNic(nicName, brName string) (int, error) {
+func configProviderNic(nicName, brName string, exchangeLinkName bool) (int, error) {
```

Comment:
There is no need to add a new parameter.

Reply:
ok， i'm trying to fix a issue. talk with you later